src/Compiler/CompilerOptions.cpp

```diff
@@ -106,12 +106,14 @@ llvm::cl::opt<bool> VerboseOutput("v", llvm::cl::desc("Use verbose output"),
 llvm::cl::opt<std::string> Xopt("Xopt",
     llvm::cl::desc("Arguments to forward to LLVM's 'opt' option processing"),
     llvm::cl::value_desc("A valid LLVM's 'opt' option"),
-    llvm::cl::cat(OnnxMlirOptions), llvm::cl::Hidden, llvm::cl::ValueRequired);
+    llvm::cl::cat(OnnxMlirOptions), llvm::cl::Hidden, llvm::cl::ValueRequired,
+    llvm::cl::OneOrMore);
```

Comment:
Does one or more mean that at least one is required?

Reply:
Please explain the rational. Does it add to, or replace?

Reply:
It exists:
```
.. _cl::ZeroOrMore:

* The **cl::ZeroOrMore** modifier (which is the default for the `cl::list`_
  class) indicates that your program will allow the option to be specified zero
  or more times.
  ```